sdk/src/Services/DynamoDBv2/Custom/Internal/InternalConstants.cs

```diff
@@ -0,0 +1,22 @@
+﻿/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ * 
+ *  http://aws.amazon.com/apache2.0
+ * 
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+namespace Amazon.DynamoDBv2.Custom.Internal
+{
+    internal static class InternalConstants
+    {
+        internal const string RequiresUnreferencedCodeMessage = "The Amazon DynamonDB high level libraries in the DocumentModel and DataModel namespace do not supported for Native AOT.";
```

Comment:
Nit: `do not supported`. 

Did you mean `do not support Native AOT`? Or should it be `have not been updated to support Native AOT` (like the other messages)?

Reply:
Fixed